README.md

```diff
@@ -18,6 +25,18 @@ sudo gem install govuk-connect
 
 ## Usage
 
-```bash
-govuk-connect --help
+If you have the gds-cli installed (you [should do](https://docs.publishing.service.gov.uk/manual/access-aws-console.html)!), you can use this tool within it. This means you only have to use one tool to do AWS and SSH.
+
 ```
+gds govuk connect --help
```

Comment:
Since we're offering two installation methods, should we keep both usages?

Reply:
Whichever installation method you use it will still work.

Reply:
I'm a little confused. What if I do `gds-cli` and I do `gem install govuk-connect`?

Reply:
`gds` and `gds-cli` are synonymous in my head, maybe that's bad. But they both work. But I expect most people to go for the Homebrew approach as it's easier than getting weird errors about system Ruby versions.